doc/source/user_guide/visualization.rst

```diff
@@ -1496,6 +1494,30 @@ Here is an example of one way to easily plot group means with standard deviation
 
    plt.close("all")
 
+Asymmetrical error bars are also supported, however raw error values must be provided in this case. For a ``N`` length :class:`Series`, a ``2xN`` array should be provided indicating lower and upper (or left and right) errors. For a ``MxN`` :class:`DataFrame`, asymmetrical errors should be in a ``Mx2xN`` array.
+
+Here is an example of one way to plot the min/max range using asymmetrical error bars
+
+.. ipython:: python
+
+   mins = gp3.min()
+   maxs = gp3.max()
+   mins
+   maxs
```

Comment:
It doesn't seem to me like these values need to be displayed. Do they make the example more clear?

Reply:
indeed not really necessary, removed